src/conditional.test.ts

```diff
@@ -68,42 +72,177 @@ describe("runtime (dataLast)", () => {
 });
 
 describe("typing", () => {
-  it("can narrow types in the transformers", () => {
-    const data = 3 as number | string;
-    conditional(
-      data,
-      [
-        isString,
-        (str) => {
-          expectTypeOf(str).toEqualTypeOf<string>();
-          expectTypeOf(str).not.toEqualTypeOf<number>();
```

Comment:
I removed `expectTypeOf(str).not.toEqualTypeOf` since there is no need to check both statements because `toEqualTypeOf` checks for the exact type.

Reply:
**praise:** agreed!